### DISCUSSION TITLE
can you use dask to analyze pdf content?

### DISCUSSION THREAD

**Discussion by abcnow** [dask/dask#10650] [General]
New here.. just checking if you can use dask to analyze pdf content (texts, images, etc).. and divide them into chunks etc

**Comment by fjetter** [comment]
Of course, you can! 
https://github.com/mrocklin/arxiv-matplotlib is an example that uses dask on [coiled](https://www.coiled.io/) to analyse pdfs on [arXive.org](https://arxiv.org/). There is a jupyter notebook and you can swap out the function that analyses the file with whatever you have in mind.
